indico/modules/events/registration/util.py

```diff
@@ -76,17 +76,24 @@ class ActionMenuEntry:
     extra_classes: str = ''
 
 
-def import_user_records_from_csv(fileobj, columns, delimiter=','):
+def import_user_records_from_csv(fileobj, columns, delimiter=None):
     """Parse and do basic validation of user data from a CSV file.
 
     :param fileobj: the CSV file to be read.
     :param columns: A list of column names, 'first_name', 'last_name', & 'email' are compulsory.
-    :param delimiter: the CSV separator.
+    :param delimiter: the CSV separator. Guessed if omitted.
     :return: A list dictionaries each representing one row,
              the keys of which are given by the column names.
     """
     with csv_text_io_wrapper(fileobj) as ftxt:
-        reader = csv.reader(ftxt.read().splitlines(), delimiter=delimiter)
+        content = ftxt.read().splitlines()
+    if not delimiter and content:
+        delimiters = [d.delimiter for d in CSVFieldDelimiter]
```

Comment:
Is this actually needed? The only thing this avoids is allowing `:` as a delimiter, but I don't see any reason where this would be an issue here.